docs/preview_conversion_analyzer.md

```diff
@@ -0,0 +1,129 @@
+<!-- The purpose of this spec is to describe a new feature and
```

Comment:
Could you delete these template comments?

Reply:
Removed Comments